src/patternfly/components/ToggleGroup/toggle-group.scss

```diff
@@ -0,0 +1,70 @@
+.pf-c-toggle-group {
+  // button
+  --pf-c-toggle-group__button--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-toggle-group__button--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-toggle-group__button--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-toggle-group__button--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-toggle-group__button--first-child--BorderTopLeftRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-toggle-group__button--first-child--BorderBottomLeftRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-toggle-group__button--last-child--BorderTopRightRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-toggle-group__button--last-child--BorderBottomRightRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-toggle-group__button--BackgroundColor: var(--pf-global--BackgroundColor--200);
+  --pf-c-toggle-group__button--Color: var(--pf-global--Color--100);
+  --pf-c-toggle-group__button--m-light--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-toggle-group__button--hover--BackgroundColor: var(--pf-global--disabled-color--200);
```

Comment:
Hmm... I don't think we should use `--disabled-color--200` here. Can we confirm we can't use one of the existing background colors? Maybe we need a new global background color var? cc @mceledonia, this ties into the theming. If someone were to customize the `--disabled-color--200` var, I don't think we would want to change this button state color.

https://github.com/patternfly/patternfly/blob/2167795ab3c6d711837f96c1f76e870fd041e3ec/src/patternfly/sass-utilities/scss-variables.scss#L25-L41